cmake/defaults/CXXDefaults.cmake

```diff
@@ -82,6 +82,11 @@ else()
     set(PXR_PYTHON_SUPPORT_ENABLED "0")
 endif()
 
+# Set plugin path environment variable name
+if (PXR_PLUGINPATH_NAME)
```

Comment:
- Did you mean for this (and below) to be PXR_OVERRIDE_PLUGINPATH_NAME?

- I think we should use the _add_define helper here if we can, something like:

if (PXR_OVERRIDE_PLUGINPATH_NAME)
    _add_define(PXR_PLUGINPATH_NAME=${PXR_OVERRIDE_PLUGINPATH_NAME})
endif()

- Does this work properly if PXR_OVERRIDE_PLUGINPATH_NAME is set to a path that has whitespace in it, like "/tmp/USD Plugins" or something like that? I'm wondering if we might have to put quotes around the ${PXR_OVERRIDE_PLUGINPATH_NAME} above.

- For organization, could you move this block of code up to where other defines are added, around line 50?

Reply:
Hi, yes, you're totally right about that (at first I was planning to use the `PXR_PLUGINPATH_NAME` variable as the option name but I thought that would be a bit missleading, thus the `OVERRIDE` name for the option)
And the weird thing is: it seems to work as it is (that's why I didn't notice that I forgot to rename those) Not sure if my setup is broken or if it's a mystery of CMake.

Anyway to answer your points:

* totally right, as I said, a mistake, I'll change that
* noted, I'll use `_add_define`
* don't know. I'll have to check (I don't know how CMake behaves in this case)
* sure

Reply:
Re: the spaces in the env setting name -- of course, I totally misread that. Thanks!